pom.xml

```diff
@@ -414,11 +383,6 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <source>${maven.compiler.source}</source>
-          <target>${maven.compiler.source}</target>
```

Comment:
I think it would still be better to use a single property. I'm going to introduce a `java.version` property in the style of Spring Boot